### PR TITLE
Feat playlist

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-4-23 (1)

### CODE REVIEW

File: video-app/src/Component/PlaylistModal/PlaylistModal.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.Modal{
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+     height: 100%;
+    background-color: #00000099;
```
Comment (arpit01923) [inline video-app/src/Component/PlaylistModal/PlaylistModal.css:7]: use it in root

**Review by arpit01923 — APPROVED** [review]
- clean code
- good naming convention
- left suggestion and try barrel exports and absolute paths

**Review by SanketDhabarde — APPROVED** [review]
- Code looks good
- Functionality working as expected
- Like the use of navigate

File: video-app/src/Component/PlaylistModal/PlaylistModal.css (hunk 1 above)
Comment (aniketsingh13) [reply video-app/src/Component/PlaylistModal/PlaylistModal.css:7]: will do that

**Pull request merged**